docs/src/main/tut/optics/lens.md

```diff
@@ -101,6 +101,41 @@ val address = GenLens[Person](_.address)
 (address composeLens streetNumber).get(john)
 (address composeLens streetNumber).set(2)(john)
 ```
+### Other Ways of Lens Composition
+
+Is possible to compose few `Lenses` together by using `compose`:
+
+```tut:silent
+GenLens[Person](_.name).set("Mike") compose
+GenLens[Person](_.age).modify(_ + 1)
+```
+
+Same but with the simplified macro based syntax:
+
+```tut:silent
+import monocle.macros.syntax.lens.
+
+john
+  .lens(_.name).set("Mike") compose
+  .lens(_.age).modify(_ + 1)
+```
+
+Sometimes you need an easy way to update `Product` type inside
+`Sum` type - for that case you can compose `Prism` with `Lens` by using `some`:   
+
+```tut:silent
+import monocle.std.option.some
+
+case class A(b: Option[B])
+case class B(c: Int)
+
+val _b = GenLens[A](_.b)
+val _c = GenLens[Option[B]](_.map(_.c))
```

Comment:
GenLens only work for case classes and fields of a case class, so it would not compile normally